python/src/FieldFunctionImplementation_doc.i.in

```diff
@@ -1,82 +1,98 @@
 %define OT_FieldFunction_doc
-"Field function.
+"Function mapping a field to a field.
 
 Available constructors:
-    FieldFunction(*meshDimension=1*)
-
-    FieldFunction(*function, meshDimension=1*)
+    FieldFunction(*spatialDim, inputDim, outputDim*)
 
 Parameters
 ----------
-function : :class:`~openturns.Function`
-    Function :math:`g: \Rset^d \mapsto \Rset^q` used to define a
-    :class:`~openturns.ValueFunction` object.
-meshDimension : int, :math:`n \geq 0`
-    Dimension of the mesh :math:`\cM`.
+spatialDim : int, :math:`\geq 1`
+    Dimension :math:`n` of the domain :math:`\cD`
+inputDim : int, :math:`\geq 1`
+    Dimension :math:`d` of the input field values 
+outputDim : int, :math:`\geq 1`
+    Dimension :math:`d'` of the output field values 
 
 Notes
 -----
-Field functions are functions being able to act on fields.
-Two particular field functions are defined:
-the :class:`spatial function <openturns.ValueFunction>` and
-the :class:`temporal function <openturns.VertexValueFunction>`.
+Field functions act on fields to produce fields:
 
+.. math::
 
-A field function
-:math:`f_{dyn}:\cD \times \Rset^d \mapsto \cD' \times \Rset^q`
-where :math:`\cD \in \Rset^n` and :math:`\cD' \in \Rset^p` is defined by:
+    f: \left| \begin{array}{rcl}
+                \cM_N \times (\Rset^d)^N & \rightarrow & \cM_{N'} \times (\Rset^{d'})^{N'} \\
+                F & \mapsto & F'
+              \end{array} \right.
 
-.. math::
+with  :math:`\cM_N` a mesh of :math:`\cD \subset \Rset^n`,  :math:`\cM_{N'}` a mesh of :math:`\cD' \subset \Rset^{n'}`.
 
-    f_{dyn}(\vect{t}, \vect{x}) = (t'(\vect{t}), v'(\vect{t}, \vect{x}))
+A field is represented by a collection :math:`(\vect{t}_i, \vect{v}_i)_{1 \leq i \leq N}` of elements of :math:`\cM_N \times (\Rset^d)^N` where :math:`\vect{t}_i` is a vertex of :math:`\cM_N` and :math:`\vect{v}_i` the associated value in :math:`\Rset^d`.
 
-with :math:`t': \cD \mapsto \cD'` and
-:math:`v': \cD \times \Rset^d \mapsto \Rset^q`.
+The constructor builds an object which evaluation operator is not defined (it throws a *NotYetImplementedException*). The instanciation of such an object is used to extract an actual :class:`~openturns.FieldFunction` from a  :class:`~openturns.Study`.
 
-A field function :math:`f_{dyn}` transforms a multivariate stochastic
-process:
+See also
+--------
+PythonFieldFunction, OpenTURNSPythonFieldFunction 
 
-.. math::
 
-    X: \Omega \times \cD \mapsto \Rset^d
+Examples
+--------
+>>> import openturns as ot
 
-where :math:`\cD \in \Rset^n` is discretized according to the mesh :math:`\cM`
-into the multivariate stochastic process:
+Using the class :class:`~openturns.OpenTURNSPythonFieldFunction` allows to define a persistent state between the evaluations of the function.
 
-.. math::
+>>> class FUNC(ot.OpenTURNSPythonFieldFunction):
 
-    Y=f_{dyn}(X)
+>>>     def __init__(self):
```

Comment:
use continuation dots ... instead of >>> for a multiline statement like this class
there are several of these in your new doc, plus myPyFunc